resources/js/components/Home/AuthBanner.vue

```diff
@@ -13,35 +20,32 @@ onMounted(() => window.addEventListener('scroll', setBannerVisibility))
 onUnmounted(() => window.removeEventListener('scroll', setBannerVisibility))
 
 function gotoAuthSection() {
-  window.scrollTo({
-    top: window.innerHeight,
-    behavior: 'smooth',
-  })
+  emit('scrollToAuth', isLoginRef.value)
 }
 
 function gotoRegisterSection() {
+  isLoginRef.value = false
   gotoAuthSection()
-  if (isLoginRef)
-    isLoginRef.value.isLogin = false
 }
 
 function gotoLoginSection() {
+  isLoginRef.value = true
   gotoAuthSection()
-  if (isLoginRef)
-    isLoginRef.value.isLogin = true
 }
 
 </script>
 
 <template>
-  <div>
-    <div class="h-16 lg:hidden" />
-    <div :class="{ '-translate-y-full': isHidden }"
-         class="lg:hidden top-0 duration-200 justify-evenly z-10 backdrop-blur-md bg-white/30 lg:bg-white fixed w-full flex items-center gap-x-6 px-6 py-2.5"
+  <div class="flex h-fit">
+    <div :style="{height: `${height}px`}" class="lg:hidden" />
+    <div ref="banner" :class="{ '-translate-y-full': isHidden }"
+         class="lg:hidden flex-1 duration-200 justify-evenly z-10 backdrop-blur-md bg-white/30 lg:bg-white fixed w-full flex items-center gap-x-6 px-6 py-2.5"
     >
       <div class="justify-center flex-1 hidden lg:flex" />
       <div class="hidden lg:justify-center flex-1 xs:flex">
-        <p class="font-bold text-sm leading-6 text-gray-900">Testy</p>
+        <svg class="size-10" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Why you dont use icon from `"@heroicons/vue"` ?

Reply:
Because that's our custom logo